src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.String.cs

```diff
@@ -35,20 +32,21 @@ private void WriteStringValueHelper(ReadOnlySpan<byte> utf8Value)
         /// <summary>
         /// Writes the string text value (as a JSON string) as an element of a JSON array.
         /// </summary>
-        /// <param name="value">The value to be written as a UTF-8 transcoded JSON string element of a JSON array.</param>
-        /// <remarks>
-        /// <para>The value is escaped before writing.</para>
-        /// <para>
-        /// If <paramref name="value"/> is <see langword="null"/> the JSON null value is written,
-        /// as if <see cref="WriteNullValue"/> was called.
-        /// </para>
-        /// </remarks>
+        /// <param name="value">The value to write.</param>
         /// <exception cref="ArgumentException">
         /// Thrown when the specified value is too large.
         /// </exception>
         /// <exception cref="InvalidOperationException">
         /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
         /// </exception>
+        /// <remarks>
```

Comment:
Out of curiosity, why did we re-order the tags (remarks/exception)? Was it for consistency and is that the expected ordering we should always follow?

Reply:
I reorder to be consistent with what we had in other places... plus I find it easier to have the remarks always at the end. //cc @carlossanlop @mairaw does the order here matter?

Reply:
It doesn't matter here. When you build corefx, the xmldoc files will have their items generated in the same order, always.

Reply:
No, the order doesn't matter,

Reply:
Plus, the porting tool only cares that the element exists, not its location.